packages/data-service-generator/tests/e2e/data-service-generator.e2e.spec.ts

```diff
@@ -193,8 +192,54 @@ describe("Data Service Generator", () => {
             logger.error(err.message, err);
           });
 
+        // Wait for the server to be ready and for database migration before running tests.
+        // TODO replace with a better docker-compose solution that waits for the migraton to be completd before starting the server
+        logger.info("Waiting for db migration to be ...");
```

Comment:
"to be ? ... " :) Did you mean "to be completed..." ?

Reply:
fixed